config/lib/mongoose.js

```diff
@@ -29,6 +29,8 @@ module.exports.connect = function (cb) {
       console.log(err);
     } else {
 
+      mongoose.Promise = config.db.promise;
```

Comment:
I debated that and since things have been moving to be as configurable as possible, I figured I'd follow that trend. I'm not opposed to changing it though. 

You could in theory set it to be bluebird or q or any other promise library.